### PR TITLE
feat: Add navigation component

### PR DESCRIPTION
**Pull request by Meghana-sk**
- Added navigation component desktop version only
- - csb link: https://7uov1u.csb.app/components/navbar/navbar.html

### CODE REVIEW

**Review by aamir74 — APPROVED** [review]
UI looks good.
Code is clean.

File: components/docs.css [hunk 1]
```diff
@@ -1,13 +1,6 @@
-* {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
+:root {
+  --br-rad-s: 8px;
```
Comment (Abhijeet43) [inline components/docs.css]: class naming can be better and more descriptive

File: components/docs.css [hunk 2]
```diff
@@ -1,13 +1,6 @@
...
 }
-
-html {
-  font-size: 16px;
-}
-
 .nav-section {
   background-color: white;
```
Comment (Abhijeet43) [inline components/docs.css]: You can create a variable for this color as well...As  you have created variables for other color variants

**Review by Abhijeet43 — APPROVED** [review]
- The UI looks good and clean
- The directory structure is also well managed
- Code is well formatted
- Suggestions: Try using variables for colors, Avoid using color names try using hex or rgb for colors.. Some class namings can be improved

**Pull request merged**